### PR TITLE
record failed lookups

### PR DESCRIPTION
**Pull request by baraabourghli**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -46,6 +46,7 @@ group :development do
   gem 'capistrano'
   gem 'capistrano-rails'
   gem 'capistrano-bundler'
+  gem 'rubocop', require: false
```
Comment (baraabourghli) [inline Gemfile]: If you don't mind adding it.

Comment (calumhalcrow) [inline Gemfile]: Yeah, that's totally fine.

**Comment by calumhalcrow** [comment]
Glad to see you found my previous work on this. TBH, I forgot that I had started it :-)

I see you've marked this as "do not merge", so I'll wait until you've done more work before giving feedback.

**Comment by baraabourghli** [comment]
I am not sure about https://github.com/GovWizely/industry_mapping/commit/7f3125a92e3cd5477498894e99bdaa85241af4f4

Let me know so I remove it if you think that it is too much.

**Comment by baraabourghli** [comment]
I deleted the Rubocop changes, I think it might be better to do it if we have more specs in this project.

**Pull request merged**